Report/create_reports.R

```diff
@@ -32,20 +32,30 @@ state_geos = locations %>%
                 filter(nchar(.data$geo_value) == 2) %>%
                 pull(.data$geo_value)
 signals = c("confirmed_incidence_num",
-            "deaths_incidence_num")
+            "deaths_incidence_num",
+            "confirmed_admissions_covid_1d")
 
 predictions_cards = get_covidhub_predictions(forecasters,
                                              signal = signals,
+                                             ahead = 1:28,
                                              geo_values = state_geos,
                                              verbose = TRUE,
-                                             use_disk = TRUE)
+                                             use_disk = TRUE) %>% 
+    filter(!(incidence_period == "epiweek" & ahead > 4))
```

Comment:
Just curious about this addition. It looks like this wasn't here before, yet we were still only saving aheads 1-4 for epiweek predictions (cases and deaths). I thought Jed made this cutoff elsewhere.

Reply:
By default, `get_covidhub_predictions` uses `ahead = 1:4` for both day and epiweek forecasts. However, daily forecasts actually go up to aheads of 28. To get those without getting epiweek forecasts more than 4 weeks ahead, I switched the `ahead` setting to 1-28 and added the filter.

We could do two separate calls to `get_covidhub_predictions` here, one for cases + deaths and one for hospitalizations, with different `ahead` settings. However the underlying [`get_forecaster_predictions_alt` downloads all forecast files](https://github.com/cmu-delphi/covidcast/blob/d34272dd3b0577b11833c696ed87b13316aca535/R-packages/evalcast/R/get_covidhub_predictions.R#L333) every time it's run (are you aware of any particular reason for this?), so the memory/speed tradeoff is poor.

Reply:
> are you aware of any particular reason for this?

Ah, perhaps because this was originally intended to be run in the GitHub Actions, the files wouldn't persist between sessions anyway.

Reply:
I see, that makes sense. And yes I believe that is the case re: downloading files.